portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,159 @@
+                        <div class="tm-timeline-connector-vertical"></div>
+                    </div>
+
+                    <div class="tm-timeline-item">
+                        <div class="tm-timeline-item-inner">
+                            <img src="images/embroidery.jpg" alt="Image" class="rounded-circle tm-img-timeline">
+                            <div class="tm-timeline-connector">
+                                <p class="mb-0">&nbsp;</p>
+                            </div>
+                            <div class="tm-timeline-description-wrap">
+                                <div class="tm-bg-dark-light tm-timeline-description">
+                                    <h3 class="tm-text-orange tm-font-400">Embroidery</h3>
+                                    <p>There's no real backstory or reason behind this one, it just seemed therapeutic and fun.</p>
+                                    <p class="tm-text-orange float-right mb-0">2020</p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="tm-timeline-connector-vertical"></div>
+                    </div>
+
+                    <div class="tm-timeline-item">
+                        <div class="tm-timeline-item-inner">
+                            <img src="images/code.jpg" alt="Image" class="rounded-circle tm-img-timeline">
+                            <div class="tm-timeline-connector">
+                                <p class="mb-0">&nbsp;</p>
+                            </div>
+                            <div class="tm-timeline-description-wrap">
+                                <div class="tm-bg-dark tm-timeline-description">
+                                    <h3 class="tm-text-green tm-font-400">Coding</h3>
+                                    <p>I'd actually never written a proper line of code before University! I consider myself a beginner with lots to learn now, and I'm sure I'll never run out of things to learn in Computer Science.</p>
+                                    <p class="tm-text-green float-right mb-0">2018 - </p>
+                                </div>
+                            </div>
+                        </div>
+
+                    </div>
+
+                </div>
+            </div>
+            <hr>
+        <!-- Footer -->
+        <footer class="row mt-5 mb-5">
+            <div class="col-lg-12">
+                <p class="text-center tm-text-gray tm-copyright-text mb-0">Copyright &copy;
+                    <span class="tm-current-year">2020</span> Michaela 
+                    
+                    | Design: Templatemo
+                </p>
+            </div>
+        </footer>
     </div>
-  </body>
-</html>
+    <!-- .container -->
+
+    <script src="scripts/jquery.min.js"></script>
+    <!-- <script src="js/jquery-3.2.1.slim.min.js"></script> -->
+    <script src="scripts/templatemo-script.js"></script>
+
+</body>
+</html>
```

Comment:
usually it is a convention to add a new line at the end of the file

Reply:
I see, thank you so much for the advice! I'll take note of that in the future.